README.md

```diff
@@ -1,97 +1,41 @@
-# Getting Started with ASP.NET Core
 
 [![Join the chat at https://gitter.im/aspnet/Home](https://badges.gitter.im/aspnet/Home.svg)](https://gitter.im/aspnet/Home?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-This guide is designed to get you started building applications with the latest development versions ASP.NET Core. This means nightly builds and potentially broken or unstable packages.
+**This is not the homepage for ASP.NET Core.** This is the home page of our source code repositories and is intended for those contributing to ASP.NET Core or using bleeding edge nightly builds.
 
-If you want a more stable, released, experience then you should go to https://www.asp.net/vnext.
+If you want a more stable, released, experience or getting started instructions then go to one of the following:
 
-## What you need
+- [.NET Landing Page](http://dot.net)

```

Comment:
@glennc If I made you laugh, my pleasure. Otherwise, well... at least I had a giggle. :stuck_out_tongue: